examples/drag_and_drop/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <title>Drag and Drop example</title>
+</head>
+
+<body>
+  <section id="app"></section>
+  <script type="module">
+    import init from '/pkg/package.js';
+    init('/pkg/package_bg.wasm');
+  </script>
+</body>
+
+</html>
```

Comment:
Install a plugin for your IDE or configure it to add a new line automatically at the end of all files.

Example in VS Code:
![image](https://user-images.githubusercontent.com/18517402/108104934-3c0ab880-708c-11eb-8dee-b954bda823a1.png)

Reply:
Thanks you for the feedback

In Jetbrain I needed to go  there  Settings → Editor → General → Ensure line feed at file end on Save :+1: